apps/ui/tests/features/feature-deep-link.spec.ts

```
@@ -0,0 +1,176 @@
+  handleLoginScreenIfPresent,
+  waitForAgentOutputModal,
+  getOutputModalDescription,
+} from '../utils';
+
+const TEST_TEMP_DIR = createTempDirPath('feature-deep-link-test');
+
+test.describe('Feature Deep Link', () => {
+  let projectPath: string;
+  let projectName: string;
+
+  test.beforeEach(async ({}, testInfo) => {
```

Biome (2.4.4)

[error] Unexpected empty object pattern.

(lint/correctness/noEmptyPattern)